flight_computer/modules/altimeter/altimeter.py

```diff
@@ -0,0 +1,70 @@
+from machine import I2C, Pin, SPI
+from time import sleep
+import uos
+import bmp280
+import sdcard
+
+# BOARD DEFNS
+I2C_SDA_PIN = 6
```

Comment:
i think u misunderstood and changed to I2C1 instead. you can keep I2C to gpio 0 and 1, and use SPI1 from GPIO 8-11